ush/python_utils/environment.py

```diff
@@ -111,13 +109,14 @@ def type_to_str(v):
 
 
 def list_to_str(v, oneline=False):
-    """Given a string or list of string, construct a string
-    to be used on right hand side of shell environement variables
+    """Given a string or list of strings, constructs a string
+    to be used on right hand side of shell environment variables.
 
     Args:
-        v: a string/number, list of strings/numbers, or null string('')
+        v: a string/number, list of strings/numbers, or null string(``''``)
+        oneline (bool): If the string is a single line (True) or multiple (False) ?
     Returns:
-        A string
+        a string
```

Comment:
Should the a be capitalized?

```suggestion
        A string
```